gnovm/cmd/gno/test.go

```diff
@@ -205,7 +206,7 @@ func execTest(cfg *testCfg, args []string, io commands.IO) error {
 			io.ErrPrintfln("FAIL")
 			testErrCount++
 		} else {
-			io.ErrPrintfln("ok      %s \t%s", pkg.Dir, dstr)
+			io.ErrPrintfln("ok      %s \ttotal gas used: %d", pkg.Dir, gasUsed)
```

Comment:
Why can't we have both gas used and duration? Both seem useful.

Reply:
I see that people are not often concern about that time so much.

Reply:
Maybe I'll ask in another way -- how do we benefit from removing it? If someone doesn't care about it then they don't have to use it.  If someone does care about it then they have it. It's only a few more characters being printed.

Reply:
@deelawn 
I am fine with keeping the duration to be displayed.
But I have another view, should we move duration to `-print-runtime-metrics` which currently displaying `runtime: cycle=%s imports=%d allocs=%s`?

Reply:
Before these changes, only subpackage level test execution displayed duration.  The runtime metrics are something different.  I'm only suggesting that we don't remove existing functionality if there is no good reason to do so.  I don't think duration needs to be added anywhere else it doesn't currently exist.

Reply:
You are right 👍 . I re-added the executing duration back again.
The result now looks like this:
![image](https://github.com/user-attachments/assets/18c42279-4e60-4020-ab36-7d79055ff383)